components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/responsemode/provider/AbstractResponseModeProvider.java

```diff
@@ -0,0 +1,51 @@
+/*
```

Comment:
Check the license header in all newly added files and update as suggested below.

Reply:
Updated